arbitrator/include/arbitrator_state_machine.hpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#ifndef __ARBITRATOR_STATE_MACHINE_HPP__
+#define __ARBITRATOR_STATE_MACHINE_HPP__
+
+#include <vector>
+
+namespace arbitrator
+{
+    /**
+     * Possible states for the Arbitrator to be in:
+     * INITIAL- Startup state while the rest of the system is initalizing
+     * PLANNING - Actiively generating a maneuver plan
+     * WAITING - After the completion of a plan, waiting for a timer trigger to replan
+     * PAUSED - Similar to waiting, but will ignore a timer trigger. Must be resumed externally.
+     */
+    enum ArbitratorState {
+        INITIAL,
```

Comment:
It would be more clear if you mark the first ENUM element as zero ```INITIAL = 0,```.

Reply:
I'm pretty sure this is implied, but I'll go ahead and specify it anyway.